src/CoreWCF.Primitives/src/CoreWCF/Description/CustomAttributeProvider.cs

```diff
@@ -246,15 +254,28 @@ private static MessageBodyMemberAttribute ConvertFromServiceModelMessageBodyMemb
 
         private static MessagePropertyAttribute ConvertFromServiceModelMessagePropertyAttribute(object attr)
         {
-            var messsageProperty = new MessagePropertyAttribute();
+            var messageProperty = new MessagePropertyAttribute();
             string tmpStr = GetProperty<string>(attr, nameof(MessagePropertyAttribute.Name));
             if (!string.IsNullOrEmpty(tmpStr))
             {
-                messsageProperty.Name = tmpStr;
+                messageProperty.Name = tmpStr;
             }
 
-            return messsageProperty;
+            return messageProperty;
         }
+
+        private static MessageParameterAttribute ConvertFromServiceModelMessageParameterAttribute(object attr)
+        {
+            var messageParameter = new MessageParameterAttribute();
+            string tmpStr = GetProperty<string>(attr, nameof(MessageParameterAttribute.Name));
+            if (!string.IsNullOrEmpty(tmpStr))
+            {
+                messageParameter.Name = tmpStr;
+            }
+
```

Comment:
--nit-- empty line

Reply:
There should be an empty line after a closing brace of an if block